src/libLookup/Lookup.cpp

```diff
@@ -2143,6 +2144,11 @@ void Lookup::CommitTxBlocks(const vector<TxBlock>& txBlocks) {
       return;
     }
   }
+  if (LOOKUP_NODE_MODE) {
+    std::unique_lock<shared_timed_mutex> lock(
```

Comment:
For easier and safer usage, can you create a function to do both the mutex acquisition and list clearing?